Dockerfile

```diff
@@ -44,7 +44,8 @@ COPY apps/${APP} ./apps/${APP}
 RUN pnpm install --recursive --offline --frozen-lockfile
 
 ENV NEXT_TELEMETRY_DISABLED=${NEXT_TELEMETRY_DISABLED} \
-    PROJECT_ROOT=${PROJECT_ROOT}
+    PROJECT_ROOT=${PROJECT_ROOT} \
+    GHOST_API_KEY=${GHOST_API_KEY}
```

Comment:
It is needed by `/contant` page @esirK. We probably don't need any of the `ENV` at the **builder** stage since all we're doing is `next build` and this _should_ be able to use `ARG`.

We need `ENV` in the **runner** stage because we want the ability to customize/override env vars used by `next start` via `dokku config:set`.

This seems like a good [read ](https://www.saltycrane.com/blog/2021/04/buildtime-vs-runtime-environment-variables-nextjs-docker/) & we should find some time to confirm if true.